src/shared/consts.tsx

```diff
@@ -194,6 +194,7 @@ export const enum ConditionalLogicMatch {
 }
 
 export const ALLOWED_AUDIO_FILE_TYPES = '.mp3,.wav';
```

Comment:
could you cast ALLOWED_AUDIO_FILE_TYPES from VALID_AUDIO_FILE_TYPES:
```suggestion
export const ALLOWED_AUDIO_FILE_TYPES = ALLOWED_AUDIO_FILE_TYPES .join(',');
```